docs/editor_manual/finding_your_way_around/the_dashboard.rst

```diff
@@ -9,6 +9,13 @@ The Dashboard provides information on:
 * Any pages currently awaiting moderation (if you have these privileges)
 * Your most recently edited pages
 
+In your settings file, add ``WAGTAILADMIN_RECENT_EDITS_LIMIT`` to change the number of items shown at recent edit.
```

Comment:
Can we move this comment over to e.g. http://docs.wagtail.io/en/v1.9/advanced_topics/settings.html. This documentation is for editors and the setting should only be mentioned for developers.